README.md

```diff
@@ -31,6 +31,7 @@ Prerequisites
 -------------
 
 - [PHP](http://php.net) 7.0+
+- [GIT](https://git-scm.com/) 2.6+
```

Comment:
OMG. You're late :trollface: 

```sh
$ git --version
git version 2.9.3
```

https://github.com/git/git/releases

Not sure it worth be as accurate then. Git 2+ should be enough (until someone reports any issue justifying it).